package.json

```diff
@@ -48,6 +48,6 @@
     "oae-jitsi-widgets"
   ],
   "engines": {
-    "node": "6.x"
+    "node": "6.11.1"
```

Comment:
Same comment as in Hilary - we want later Node versions to continue working so needs a `>=`.